recipes/portaudio/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "190600_20161030" %}
+
+package:
+  name: portaudio
+  version:  {{ version }}
+
+source:
+  url: http://www.portaudio.com/archives/pa_stable_v{{ version }}.tgz
+  sha256: f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - posix  # [win]
+    - m2w64-toolchain  # [win]
```

Comment:
Could you please add `alsa-lib` as both a build and run dependency, so that this package doesn't break on Linux, the same way the defautls version does?

Reply:
Sure.